eslint.config.js

```diff
@@ -0,0 +1,202 @@
+import importPlugin from 'eslint-plugin-import';
+import perfectionist from 'eslint-plugin-perfectionist';
+import prettier from 'eslint-plugin-prettier';
+import react from 'eslint-plugin-react';
+import reactHooks from 'eslint-plugin-react-hooks';
+import reactRefresh from 'eslint-plugin-react-refresh';
+import simpleImportSort from 'eslint-plugin-simple-import-sort';
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+import cspellConfigs from '@cspell/eslint-plugin/configs';
+import { join, dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+import eslint from '@eslint/js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
```

Comment:
its needed? this is made iirc by upgrade of eslint 9 , but I don't sure we need it

Reply:
yes to load the cspell exceptions